clients/unshaded/src/test/java/tachyon/client/file/options/MkdirOptionsTest.java

```diff
@@ -30,14 +30,12 @@
   @Test
   public void builderTest() {
     Random random = new Random();
+    boolean allowExists = random.nextBoolean();
     boolean recursive = random.nextBoolean();
     UnderStorageType ufsType = UnderStorageType.SYNC_PERSIST;
 
-    MkdirOptions options =
```

Comment:
nit: the individual setters were on separate lines on purpose here to improve readability